### ISSUE TITLE
Import the same solution twice

### ISSUE THREAD

**Issue by alexandrebakercode** [MicrosoftDocs/power-automate-docs#792]
Hi,

For the note at the bottom you can import the same solution to update the version. Note that it should be a greater version number and they both have to be managed or unmanaged.

Thanks,
Alex

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7d865166-93b6-e99e-9b76-13144fe77a10
* Version Independent ID: 2c16aac3-0db9-9b47-75b5-a4015eb6da7f
* Content: [Learn how to import a solution - Power Automate](https://docs.microsoft.com/en-us/power-automate/import-flow-solution)
* Content Source: [articles/import-flow-solution.md](https://github.com/MicrosoftDocs/power-automate-docs/blob/main/articles/import-flow-solution.md)
* Service: **power-automate**
* Sub-service: **cloud-flow**
* GitHub Login: @MSFTMan
* Microsoft Alias: **deonhe**

**Comment by MSFTMan** [comment]
@alexandrebakercode Thanks or reporting this issue. We will investigate this issue and update the docs as needed.

**Comment by MSFTMan** [comment]
@ChrisGarty Can you please confirm if this proposed update to the docs is correct?

**Comment by ChrisGarty** [comment]
That is correct. We should remove that note and add a link to https://learn.microsoft.com/en-us/power-apps/maker/data-platform/update-solutions as the first item in the Learn More list :)
